app/templates/orders/pending.hbs

```diff
@@ -47,6 +47,9 @@
           {{paypal-button data=model.order}}
         </div>
       {{/if}}
+      {{#if isStripe}}
```

Comment:
@mrsaicharan1 You could stack this button up with the already existing `{{#if isStripe}}` condition?